doc/examples/filters/plot_tophat.py

```diff
@@ -0,0 +1,31 @@
+"""
+================================================================
+Removing small objects in grayscale images with a top hat filter
+================================================================
+
+This example shows how to remove small objects from grayscale images. The top-hat transform [1]_ is an operation that extracts small elements and details from given images. Here we use a white top-hat transform, which is defined as the difference between the input image and its (mathematical morphology) opening.
```

Comment:
This should be wrapped at 80c wide

Reply:
oh yeah, I wrote this line in the suggestion editor which is very convenient but quite rustic...